src/ruby/pb/grpc/health/v1alpha/health_services.rb

```diff
@@ -0,0 +1,28 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Btw, I think most languages use "_grpc" or "Grpc" suffix instead of "_services". Not important for this PR, but might be worth unifying that in the future.